pkg/scheduler/cache/cache.go

```diff
@@ -961,8 +961,18 @@ func (sc *SchedulerCache) processResyncTask() {
 		return
 	}
 
+	reSync := false
 	if err := sc.syncTask(task); err != nil {
 		klog.Errorf("Failed to sync pod <%v/%v>, retry it.", task.Namespace, task.Name)
+		reSync = true
+	}
+	if task.ReSyncTask != nil {
```

Comment:
How about syncTask return no error? Looks it will always run ReSyncTask callback.

Reply:
yes, `ReSyncTask callback` will always retry and run until it return no err.

Reply:
I think what we need care is `syncTask` return err but `ReSyncTask ` return no err, then `ReSyncTask` will still run every time, we can add a flag to indicate whether `ReSyncTask` is finished.

Reply:
Which scenario will this callback be used in?

Reply:
just like what I said in description, when using custom bind method, some alarm or rollback are needed when bind err.